### PR TITLE
Add python pipeline

### PR DESCRIPTION
**Pull request by pwplusnick**

### CODE REVIEW

File: docker-secret.yaml.tmpl [hunk 1]
```diff
@@ -4,7 +4,7 @@ kind: Secret
 metadata:
   name: dockerhub-user-pass
   annotations:
-    tekton.dev/docker-0: https://index.docker.io/v1/
+    tekton.dev/docker-0: https://index.docker.io/
```
Comment (pritidesai) [inline docker-secret.yaml.tmpl]: Hey @pwplusnick I am not aware of any API changes here, v1? 🤔

**Comment by pritidesai** [comment]
thanks @pwplusnick for all the changes, last thing I need is a specific Tekton Version from you so that I can add it to the documentation.

**Comment by pwplusnick** [comment]
> thanks @pwplusnick for all the changes, last thing I need is a specific Tekton Version from you so that I can add it to the documentation.

Pipeline version: v0.11.0

**Review by pritidesai — APPROVED** [review]
thanks @pwplusnick I made some additional changes to rename image name, pipelinerun file name, and updated pipelinerun with the example from OpenWhisk-test repo with the app path.

**Pull request merged**